### PR TITLE
Generate static sites per project

### PR DESCRIPTION
**Pull request by Franpastoragusti**
## Description

- Connect to stub api
- Generate static sites per project (Detail view)
- Load leaderboard from sub api
- Connection between leaderboard and detail by attestationUid

### CODE REVIEW

File: packages/nextjs/components/onchain-impact-dashboard/Leaderboard.tsx [hunk 1]
```diff
@@ -4,12 +4,24 @@ import { useState } from "react";
...
+  const [projects, setProjects] = useState<Project[]>([]);
+
+  const getProjects = async () => {
+    const response = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/stub/projects`);
+    const data: Project[] = await response.json();
+    setProjects(data);
+  };
+  useEffectOnce(() => {
+    getProjects();
+  });
```
Comment (swellander) [inline packages/nextjs/components/onchain-impact-dashboard/Leaderboard.tsx]: I'm pretty new to `useHooks-ts`, just curious is there any difference between this and 
```js
useEffect(() => {
  getProjects();
}, []);
```

File: packages/nextjs/app/page.tsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import "./page.css";
 import type { NextPage } from "next";
 import Leaderboard from "~~/components/onchain-impact-dashboard/Leaderboard";
-import { TEST_PROJECTS } from "~~/utils/onchainImpactDashboard/initialMockData";
```
Comment (swellander) [inline packages/nextjs/app/page.tsx:4]: Can we just get rid of the entire `initialMockData.ts` file now?

**Review by swellander — APPROVED** [review]
@Franpastoragusti looks great! Will be nice to have these quick detail pages

File: packages/nextjs/components/onchain-impact-dashboard/Leaderboard.tsx (hunk 1 above)
Comment (Franpastoragusti) [reply packages/nextjs/components/onchain-impact-dashboard/Leaderboard.tsx]: is the same, I have changed it but I don't know why is not reflected

File: packages/nextjs/app/page.tsx (hunk 2 above)
Comment (Franpastoragusti) [reply packages/nextjs/app/page.tsx:4]: yes, we have now the endpoints connected and the mock data on the api side

Comment (Franpastoragusti) [reply packages/nextjs/app/page.tsx:4]: also now we have data for 337 projects in the mocked one

**Review by escottalexander — APPROVED** [review]

**Pull request merged**